test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/NotExpressionTests.cs

```diff
@@ -58,15 +60,15 @@ public async Task GivenANotExpressionPattern_WhenSearched_ThenCompareResponseWit
             var healthCareServices = healthRecordIdentifiers.Where(i => i.ResourceType == ResourceType.HealthcareService.ToString());
             foreach (HealthRecordIdentifier healthCareService in healthCareServices)
```

Comment:
How is this part of the test relevant to the :not operator? Neither of the queries use it.

Reply:
Moved those tests to a new class called IdSearchTests. 
That's another side-effect bug that was somehow related to a failures in :not operator.